integrations/easyudSDKjs/main.cpp

```diff
@@ -481,6 +481,46 @@ extern "C" {
       });
   }
 
+  EMSCRIPTEN_KEEPALIVE void udSDKJS_GetCollabUsers(int additionalInfo, void (*successCB)(const char *pJSON))
+  {
+    udWorkerPool_AddTask(g_pWorkerPool, [additionalInfo, successCB](void *)
+    {
+      udJSON json;
+      udJSON temp;
+      const char *pJSON = nullptr;
+      udSceneUpdateInfo info = {};
+      if (g_pScene != nullptr)
+      {
+        if ((udResult)udScene_Update(g_pScene, &info) == udR_Success)
+        {
+          for (uint32_t i = 0; i < info.usersCount; i++)
+          {
+            temp.SetString(info.pUserList[i].pDisplayName);
+            json.Set(&temp, "collabUsers[%u].name", i);
+            temp.SetString(info.pUserList[i].pUserID);
+            json.Set(&temp, "collabUsers[%u].userID", i);
+            temp.SetString(info.pUserList[i].pSceneSessionID);
+            json.Set(&temp, "collabUsers[%u].sessionID", i);
+            json.Set("collabUsers[%u].lastUpdated = %f", i, info.pUserList[i].lastUpdated);
+
+            // Adds additional info
+            if (additionalInfo == 1) {
+              json.Set("collabUsers[%u].userFlags = %u", i, info.pUserList[i].userFlags);
```

Comment:
I was under the impression this was a list when you asked about this; the bool is useful if there's an array of values to push out, but otherwise it's overkill to prevent a simple flags variable.

Reply:
Updated incomplete block